Docs/source/pages/faq-troubleshooting.md

```diff
@@ -104,6 +104,14 @@ Add `"com.unity.inputsystem"` to your `manifest.json`, inside `testables.`
 </details>
 <br>
 
+<details>
+<summary>Building the instrumented app, I get the error: <strong>The type or namespace name 'InputTestFixture' could not be found (are you missing a using directive or an assembly reference?)</strong>, </summary>
+<br>
+If you have Addressables package included in project you, set the Addressables settings to not build addressable when building the instrumented app. This can be done in <strong>Windows->Asset Management->Addressables->Settings</strong> and you will have an option <strong>Build Addressables on Player Build</strong>. Select <strong>Do not build Addressables content on Player build</strong>
```

Comment:
It seems like you missed a word or something here: "... include in project you, set the Addressables...".